lib/renderer/init.js

```diff
@@ -121,7 +121,7 @@ if (preloadScript) {
   try {
     require(preloadScript)
   } catch (error) {
-    if (error.code === 'MODULE_NOT_FOUND') {
+    if (error.code === 'MODULE_NOT_FOUND' && error.message.indexOf(preloadScript) > -1) {
```

Comment:
Should this be `===` 0 instead of `> -1?

That way a path like `/foo/bar.js` won't incorrectly match `/prefix/foo/bar.js`.